README.md

```diff
@@ -0,0 +1,68 @@
+# complate-express
+
+[Express](http://expressjs.com) adaptor for [complate](https://complate)
+
+
+Getting Started
+---------------
+
+Install complate-express in your Express application:
+
+```
+$ npm install complate-express
+```
+
+After registering complate's middleware, use `Response#complate` for rendering:
+
+```javascript
+let express = require("express");
+let complate = require("complate-express");
+
+let app = express();
+// register complate middleware
+app.use(complate("/path/to/views.js")));
+
+app.get("/", (req, res) => {
+    res.complate("MyView", { title: "Hello World" });
+});
+```
+
+Views are typically generated from JSX modules:
+
+```jsx
+import { createElement } from "complate-stream";
+
+export function MyView({ title }) {
+    return <article>
+        <h1>{title}</h1>
+        <p>lorem ipsum dolor sit amet</p>
+    </article>;
+}
+
+export default (view, params, stream, fragment, callback) => {
+    return renderer.renderView(view, params, stream, { fragment }, callback);
+};
+```
+
+These JSX modules are then combined into a single `views.js` bundle, e.g. using
+[faucet](faucet.org) - see
```

Comment:
that URL is not correct - need to discuss with @moonglum

Reply:
faucet-pipeline.org is the correct URL

Reply:
Thanks, fixed in 0d4389a8994ab091f29d63a175257eb06e51126f.